rel-eng/atomic-enterprise-allinone.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Atomic Enterprise Master All-In-One
+Documentation=https://github.com/openshift/origin
+After=network.target
+After=etcd.service
+Before=atomic-enterprise-sdn-node.service
+Before=atomic-enterprise-sdn-master.service
+Requires=network.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-allinone
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start --master-config=${MASTER_CONFIG_FILE} --node-config=${NODE_CONFIG_FILE} $OPTIONS
```

Comment:
Yeah, but that is pretty ugly vs. creating cleanly well defined unit files for the components. 

That would trickle through installers, deployment, etc.